src/transformStateWithClones.js

```diff
@@ -8,6 +8,32 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const allStateSteps = [];
```

Comment:
не добавляй приставку all
```suggestion
  const stateSteps = [];
```